portal/src/it/resources/application.conf

```diff
@@ -42,12 +42,23 @@ database {
   }
 
   mongo {
+    user = "mongoUser"
```

Comment:
I don`t understand, why do you use static parameters in this place? We have all parameters in .env file.

Reply:
@DruzhininPavel, thank you for your review.
I'm happy to see that you follow the changes closely.

But I would like to highlight that this request was in the draft state and it had WIP (work in progress) in the title.
So, it hasn't asked for a review at that moment, I was working on it to check some things.

Moving back to your suggestion, yes, we have `.env` file with params but it's not provided for our IT tests when we run them.
These static params using just in the IT's `application.conf`, not in the main one.
I would prefer to leave it as is.

Reply:
Oops. Sorry. I didn't saw that it was draft. I'll be better in future.

Reply:
It's okay,
Thank you!